app/Http/Requests/AddTodoRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddTodoRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title' => 'required|string|max:255',
+            'description' => 'required|string|max:255',
+            'priority' => 'required',
```

Comment:
sobirzom da je ovo enum isto tako treba izvaldiirati, pogledaj `in` za request https://laravel.com/docs/5.8/validation#rule-in

Reply:
Sto se tice ovog abort-a, sluzi ukoliko korisnik nije authentifikovan da vrati 401